model/task_build_variant_names_test.go

```diff
@@ -13,11 +13,6 @@ import (
 
 func TestFindUniqueBuildVariantNamesByTask(t *testing.T) {
 	assert.NoError(t, db.ClearCollections(task.Collection, build.Collection))
```

Comment:
Seems like this no longer needs to clear the build collection since it doesn't rely on it.